.gitignore

```diff
@@ -15,6 +15,7 @@
 *.generated.yml
 coverage.out
 .python-version
+beat.db
```

Comment:
I wonder if this file should not be in `data`?

Reply:
Auditbeat writes this file to `${path.data}`.

Reply:
But that would mean the ignore path should be `data/beat.db`?

Reply:
Having a gitignore of `beat.db` covers `data/beat.db`.